scripts/i18n/create.js

```diff
@@ -0,0 +1,351 @@
+            url
+          }
+        }
+      }
+    `,
+    {
+      headers: {
+        authorization: `token ${process.env.GITHUB_ADMIN_AUTH_TOKEN}`,
+      },
+      input: {
+        name: repoName,
+        ownerId: organization.id,
+        description: `${langName} translation of Gatsbyjs.org`,
+        visibility: `PUBLIC`,
+      },
+    }
+  )
+
+  logger.debug(`successfully created repo`)
+  return createRepository.repository
+}
+
+// set permissions
+async function createBranchProtections(repo) {
+  logger.info(`creating branch protections for ${repo.name}`)
+  return await graphql(
+    `
+      mutation($input: CreateBranchProtectionRuleInput!) {
+        createBranchProtectionRule(input: $input) {
+          branchProtectionRule {
+            id
+          }
+        }
+      }
+    `,
+    {
+      headers: {
+        authorization: `token ${process.env.GITHUB_ADMIN_AUTH_TOKEN}`,
+      },
+      input: {
+        repositoryId: repo.id,
+        pattern: branch,
+        requiresCodeOwnerReviews: true,
+        dismissesStaleReviews: true,
+        requiresApprovingReviews: true,
+        requiredApprovingReviewCount: 1,
+      },
+    }
+  )
+}
+// publish issue as the bot and pin it
+async function createProgressIssue(repo, langName) {
+  // TODO dynamically generate the issue from docs content
+  logger.info(`creating progress issue for ${repo.name}`)
+  const {
+    createIssue: { issue },
+  } = await graphql(
+    `
+      mutation($input: CreateIssueInput!) {
+        createIssue(input: $input) {
+          issue {
+            id
+            url
+          }
+        }
+      }
+    `,
+    {
+      headers: {
+        authorization: `token ${process.env.GITHUB_BOT_AUTH_TOKEN}`,
+      },
+      input: {
+        title: `${langName} Translation Progress`,
+        body: issueBody,
+        repositoryId: repo.id,
+      },
+    }
+  )
+  await graphql(
+    `
+      mutation($input: PinIssueInput!) {
+        pinIssue(input: $input) {
+          issue {
+            id
+          }
+        }
+      }
+    `,
+    {
+      headers: {
+        authorization: `token ${process.env.GITHUB_ADMIN_AUTH_TOKEN}`,
+        accept: `application/vnd.github.elektra-preview+json`,
+      },
+      input: {
+        issueId: issue.id,
+      },
+    }
+  )
+  return issue
+}
+
+async function commentOnRequestIssue(
+  issueId,
+  transRepo,
+  progressIssue,
+  maintainers
+) {
+  const maintainerStr = maintainers.map(m => `@` + m).join(`, `)
+  const body = `
+Hi ${maintainerStr}!
+
+Your repository [${transRepo.name}](${transRepo.url}) has been created.
+
+Please use this [progress issue](${progressIssue.url}) to keep track of the translation progress.
+
+If you are not already a member of the [gatsbyjs GitHub organization](https://github.com/gatsbyjs), you should have received an invitation to join. You must accept the invitation to be able to review translations! If you do not see one, please let the Gatsby team know!
```

Comment:
We should be specific about how to have people get in touch with us. We could create a new email alias or something?

```suggestion
If you are not already a member of the [gatsbyjs GitHub organization](https://github.com/gatsbyjs), you should have received an invitation to join. You must accept the invitation to be able to review translations! If you do not see one, please let the Gatsby team know via email: `translations@gatsbyjs.com`
```

Reply:
I'm not sure what this should be. Maybe they should just create an issue?

Reply:
An issue would work! That sounds like a more ergonomic way of going about it (something not getting lost/buried in email as it would be more visible to team members)